Controllers/BranchesController.cs

```diff
@@ -162,6 +162,7 @@ public async Task<IActionResult> Edit(int id, BranchFormViewModel branchVM)
                         throw;
                     }
                 }
+                return RedirectToAction(nameof(Index));
             }
             branchVM.Entities = _context.Entities.ToList();
```

Comment:
maybe worth to plan for the future errors handling

Reply:
To be discussed :)